doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2353,3 +2353,48 @@ Response code:
             }
           }
         ]
+
+
+### Notify [POST /v2/op/notify{?options}]
+
+This operation is intended to consume a notification payload so that all the entity data included by such notification is persisted, overwriting if necessary.
+This operation is useful when one NGSIv2 endpoint is subscribed to another NGSIv2 endpoint (federation scenarios). 
+The request payload must be an NGSIv2 notification payload. 
+The behaviour must be exactly the same as `POST /v2/op/update` with `actionType` equals to `APPEND`. 
```

Comment:
Given the upsert operation has been added to the NGSIv2 spec, maybe this could be changed in this way:

> The behaviour must be exactly the same as `POST /v2/entites?options=upsert`.